app/views/venues/show.html.erb

```diff
@@ -0,0 +1,43 @@
+<div class="venue-events">
+  <div class="venue-event-title">
+    <h1><%= @venue.name %> Events</h1>
+  </div>
+  <div class="event-page col-lg-8 col-lg-offset-2">
+    <div class="row event-page-header event-page-inner">
+      <table class="venue-event-list">
+        <tr class="event-header">
+          <th></th>
+          <th class="event-header-date">DATE</th>
+          <th class="event-header-title">EVENT</th>
+          <th class="event-header-locations">LOCATION/TIME</th>
+        </tr>
+        <% @events.each do |event| %>
+          <tr class="event-row">
+            <td>
+              <div class="rotate event-category">
+                <p><%= event.categories.first.name %></p>
+              </div>
+            </td>
+            <td>
+              <div class="event-date-group">
+                <p class="event-month"><%= event.date.strftime("%b") %></p>
```

Comment:
good use of strftime but these would be good methods to create so that we could use for our whole application 
`def month` 
  `  date.strftime("%b")` 
`end `
etc..